scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -225,6 +227,7 @@ def build_graph(df, dataset):
     isTop = dataset.group == "Top"
     isQCDMC = dataset.group == "QCD"
     require_prompt = "tau" not in dataset.name # for muon GEN-matching   
+    storage_type=hist.storage.Weight() # need weights also for systematic histograms for extendedABCD
```

Comment:
ok, I see the reason, but here we consume again twice the memory only for the fakes (!). Can't we use the nominal stat uncertainty in the nominal histograms somehow? The stat uncertainty is practically the same for any systematic variation, and in fact I am not even sure what we store is even always correct, since in some cases it comes from the mirroring (the problem there is that we do down=nomi^2/up, and I suspect we propagate all uncertainties as if they were all independent)

Reply:
The mirroring should always happen after the fakes now I thought.  But ok it's probably also true that using the nominal statistical uncertainty would be fine.

Reply:
Ok, yes I agree